test/core/binary.wast

```diff
@@ -183,220 +183,6 @@
   "integer representation too long"
 )
 
-;; Signed LEB128 must not be overlong
-(assert_malformed
```

Comment:
This and next one duplicates at:
https://github.com/WebAssembly/spec/blob/f607ec9a776a9e3425639f3cbe9b2d96bdc16b60/test/core/binary.wast#L458

Reply:
Also https://github.com/WebAssembly/spec/blob/f607ec9a776a9e3425639f3cbe9b2d96bdc16b60/test/core/binary-leb128.wast#L481